blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DBasicMetricsRW.java

```diff
@@ -102,6 +102,9 @@ public void insertMetrics( Collection<IMetric> metrics ) throws IOException {
                 }
 
                 futures.put( locator, rawIO.insertAsync( metric ) );
+
+                // this is marking  metrics_strings & metrics_full together.
+                Instrumentation.markFullResMetricWritten();
```

Comment:
This is how its done with astyanax too. Thats why astyanax has smoother graphs compared to datastax. They just call this, when they add metrics to mutationBatch. Down the line mutationBatch.execute could fail.